igo/igod/server.go

```diff
@@ -50,8 +50,9 @@ func (s *Server) Run() {
 		}
 		w.Write(b)
 	})
+	fmt.Println(port)
```

Comment:
let's remove this for now, we probably will want more robust diagnostic logs in the future, but that can come later.